### PR TITLE
Redesign of gas customization basic tab.

### PR DESCRIPTION
**Pull request by danjm**
BEFORE:
screenshot from 2018-09-21 00-09-23

AFTER:
screenshot from 2018-10-25 01-21-37

### CODE REVIEW

File: ui/app/components/gas-customization/gas-modal-page-container/basic-tab-content/basic-tab-content.component.js [hunk 1]
```diff
@@ -14,12 +14,14 @@ export default class BasicTabContent extends Component {
...
-        <div className="basic-tab-content__title">Suggest gas fee increases</div>
+        <div className="basic-tab-content__title">Estimated Processing Times</div>
+        <div className="basic-tab-content__blurb">Select a higher gas fee to accelerate the processing of your transaction.*</div>
         <GasPriceButtonGroup
-          className="gas-price-button-group"
+          className="gas-price-button-group--alt"
           showCheck={true}
           {...this.props.gasPriceButtonGroupProps}
         />
+        <div className="basic-tab-content__footer-blurb">*Accelerating a transaction by using a higher gas price increases it’s chances of getting processed by the network faster, but it is not always guaranteed.</div>
```
Comment (alextsg) [inline ui/app/components/gas-customization/gas-modal-page-container/basic-tab-content/basic-tab-content.component.js]: increases *its chances

File: ui/app/components/gas-customization/gas-modal-page-container/basic-tab-content/index.scss [hunk 2]
```diff
@@ -1,13 +1,28 @@
 .basic-tab-content {
   display: flex;
   flex-direction: column;
-  align-items: center;
-  margin-bottom: 22px;
-  height: 291px;
+  align-items: flex-state;
```
Comment (alextsg) [inline ui/app/components/gas-customization/gas-modal-page-container/basic-tab-content/index.scss]: Should this be `flex-start` or just be removed?

File: ui/app/selectors/tests/custom-gas.test.js [hunk 3]
```diff
@@ -119,18 +122,21 @@ describe('custom-gas selectors', () => {
       {
         expectedResult: [
           {
+            labelKey: 'fastestAllCaps',
```
Comment (alextsg) [inline ui/app/selectors/tests/custom-gas.test.js]: These `AllCaps` need to be updated right?

**Review by alextsg — APPROVED** [review]

**Comment by bdresser** [comment]
looks great!!
- what happens if I switch to fiat as my "primary" currency?
- can we bump the padding up on the right side of the gray text so it doesn't stick out past the buttons, and change `it's` to `its` :)
screen shot 2018-10-24 at 10 28 07 pm

**Comment by danjm** [comment]
@bdresser 

> can we bump the padding up on the right side of the gray text so it doesn't stick out past the buttons, and change it's to its :)
This change was made

> what happens if I switch to fiat as my "primary" currency?
Good point. I am going to do another PR that applies the same sort of currency toggle changes that Alex recently made to all of the new gas customization stuff.

**Pull request merged**